src/app/Shared/Redux/Configurations/DashboardConfigSlice.tsx

```diff
@@ -45,6 +45,7 @@ export enum DashboardConfigAction {
   TEMPLATE_RENAME = 'template-config/rename',
   TEMPLATE_HISTORY_PUSH = 'template-history/push',
   TEMPLATE_HISTORY_CLEAR = 'template-history/clear',
+  CLEAR_ALL_CARDS = 'dashboard-card-config/clear-all-cards',
```

Comment:
After ensuring all functionalities work as expected, would u mind replacing `Clear All Cards` to `Clear Layout` in all variables, enums here and the ReduxStore.tsx?

Reply:
For this file, please name it as `LAYOUT_CLEAR` to follow the convention above.

Reply:
And the locale translation key: https://github.com/cryostatio/cryostat-web/pull/1078#discussion_r1293834382